.github/workflows/ci.yml

```diff
@@ -209,6 +209,13 @@ jobs:
       - name: Add sccache to path
         run: '[Environment]::SetEnvironmentVariable("PATH", [Environment]::GetEnvironmentVariable("PATH", [EnvironmentVariableTarget]::Machine) + ";" + (Split-Path -Path $env:SCCACHE_PATH -Parent), [EnvironmentVariableTarget]::Machine);'
         shell: powershell
+      - name: Install jom
+        run: |
+          Invoke-WebRequest -Uri "http://download.qt.io/official_releases/jom/jom.zip" -OutFile "jom.zip"
+          New-Item -ItemType Directory -Path D:\jom
+          Expand-Archive -Path jom.zip -DestinationPath D:\jom
+          [Environment]::SetEnvironmentVariable("PATH", [Environment]::GetEnvironmentVariable("PATH", [EnvironmentVariableTarget]::Machine) + ";D:\jom", [EnvironmentVariableTarget]::Machine);
+        shell: powershell
```

Comment:
should we verify the zip's hash like we do with sqliteodb_w64.exe?

Reply:
Good point, updated in 2ec68cae5d0af75e3dc7219a4164d3d529b6987c